hack/test-cmd.sh

```diff
@@ -190,22 +191,19 @@ runTests() {
   #######################
 
   # Make sure the UI can be proxied
-  start-proxy --api-prefix=/
+  start-proxy
   check-curl-proxy-code /ui 301
   check-curl-proxy-code /metrics 200
+  check-curl-proxy-code /api/ui 404
   if [[ -n "${version}" ]]; then
     check-curl-proxy-code /api/${version}/namespaces 200
   fi
   stop-proxy
 
-  # Default proxy locks you into the /api path (legacy behavior)
```

Comment:
Since the default value is changed, we don't lock you into `/api` by default, so remove this test.